docs/combine.rst

```diff
@@ -45,29 +45,32 @@ Options
 
         ``all``:   all of the types listed here.
 
-        ``ammo``: AMMO
+        ``ammo``: AMMO. Max 25.
 
-        ``drink``: DRINK
+        ``drink``: DRINK. No Max.
 
-        ``fat``:   GLOB and CHEESE
+        ``fat``:   GLOB and CHEESE. Max 5.
 
-        ``fish``:  FISH, FISH_RAW and EGG
+        ``fish``:  FISH, FISH_RAW and EGG. Max 5.
 
-        ``food``:  FOOD
+        ``food``:  FOOD. Max 20.
 
-        ``meat``:  MEAT
+        ``meat``:  MEAT. Max 5.
 
-        ``parts``: CORPSEPIECE
+        ``parts``: CORPSEPIECE. Max 1.
```

Comment:
Can corpsepiece be removed from this list if the max stack size is 1?

Edit: oh I see, it's still useful for -u.

Would this be more useful in general if it just handled teeth and the default stack size was large?